src/tests/functional/requirements.txt

```diff
@@ -1,7 +1,6 @@
 flake8
-# to run func tests on controllers < 3.0
 juju
 mock
 pytest
-pytest-asyncio
+pytest-asyncio<0.23
```

Comment:
Updated.

Reply:
Could we identify the breaking change more precisely ? So that we can remove this restriction one day.

Reply:
Maybe we can link to the [changelog](https://pytest-asyncio.readthedocs.io/en/latest/reference/changelog.html#id12) itself?

Reply:
Updated.

Reply:
> Instead of spending time to fix functional test for version after 0.23,
   We should spend time to follow up with pytest-operator or zaza.

I don't think we need these lines. Maybe just: `waiting for fix from zaza/ pytest-operator` is sufficient.

Reply:
We are not using zaza or pytest-operator here.